### PR TITLE
use awk instead

### PR DESCRIPTION
**Pull request by brntbeer**
Using awk instead of cut to avoid ambiguous reference. 

screen shot 2013-05-28 at 1 20 57 pm

used `cut -c1-4` to force ambiguous error on git/git project, but it can still pop up in smaller projects for the first 7 characters.

/cc @matthewmccullough to also update your dotfiles.

### CODE REVIEW

File: presentations/_posts/foundations/reflog/0001-01-01-Restoring-by-Action.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ categories: ['slidecontent']
 	* rebase
 	* reset
 
-* `gitk --all `git reflog | cut -c1-7``
+* `gitk --all `git reflog | awk '{print $1}'``
 * `git log --pretty=oneline --abbrev-commit --graph --decorate `git reflog | cut -c1-7``
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/reflog/0001-01-01-Restoring-by-Action.md]: Sweet.  Should we do the same on the `git log` call beneath it as well?

`awk` forever!

Comment (brntbeer) [inline presentations/_posts/foundations/reflog/0001-01-01-Restoring-by-Action.md]: Snap! didnt even notice that. yes. one sec

**Comment by matthewmccullough** [comment]
I'm totally happy with this more flexible non-column based change.  I wonder in the back of my mind if there's a case where `cut` is installed but `awk` is not, but I literally evicted worrying about that after 5 seconds.

Love this polish.  Thanks @brntbeer.  I'm :+1: on :shipit:

**Comment by brntbeer** [comment]
Fairly certain both are installed by default on almost all linux systems as well as Macs.

In Unix books explaining commands, Cut, Sed, and Awk chapters are almost always close to each other.

**Pull request merged**